### ISSUE TITLE
Add rolls on users

### ISSUE THREAD

**Issue by Yoshi20** [Yoshi20/FranceSmash#10]

**Comment by teluoborg** [comment]
what are rolls? simple curiosity

**Comment by Yoshi20** [comment]
Not 100% sure yet but something like:
- TO
- news editor
- community editor
- streamer
- photographer
- ...

**Comment by Yoshi20** [comment]
First Prio:
- "TO"
- "news_editor"
- "communities_editor"

**Comment by teluoborg** [comment]
Yeah I agree on the priority ones. What would the others be able to do (photographer, streamer)?

**Comment by Yoshi20** [comment]
don't know yet :D

**Comment by teluoborg** [comment]
then yeah they aren't important XD
